lib/cionic/emg_iir.h

```diff
@@ -51,5 +52,9 @@ int iir_filter_process(iir_filter_t *filter, float *norms, int numchans,
                         elapsed_t ts_in, const uint8_t *buffer,
                         elapsed_t *ts_out, float *uv_out);
 
+// generated coefficients
+extern float emg_highpass_filter_sos[NO_OF_BQS][NO_OF_COEFFS_PER_BQ];
```

Comment:
Could you please use getter/setter API instead of extern def.

Reply:
done